src/js/components/Step/Step.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import Col from '../Col/Col';
+import Icon from '../Icon/Icon';
+
+type Props = {
+  activeStep: number,
+  isLast: boolean,
+  label: string,
+  thisStep: number,
+  value: number | string,
+};
+
+const Step = ({ label, value, isLast, activeStep, thisStep }: Props) => {
+  const classes = classNames(
+    'd-flex align-items-center stepper__step mb-1',
+    { 'stepper__step--active': activeStep === thisStep },
+    { 'stepper__step--todo': thisStep > activeStep }
```

Comment:
This confused me a little, couldn't we have "todo" be the default style? Then "active" and "done" be the modifiers?